ui/src/constants/index.ts

```diff
@@ -379,3 +367,109 @@ export {
   ZONE_IMG_MAP,
   getMinResource,
 };
+
+export const STATUS_LIST = [
```

Comment:
Why no use map directly instead of list if all the usages are paired with `findByValue`?

Reply:
The data processed by the map cannot be used directly and is prone to errors.

Reply:
I think `{"running": {...}}` can be used directly. What if `findByValue` find nothing if you use list? When talking about error-proneness lists are almost the same as maps.

Reply:
Different methods, different habits, the same results （＾∀＾）